{
    "id": 113798,
    "name": "CVE-2018-1000808",
    "cwe": "CWE-404",
    "modified": "2021-08-04T15:14:46",
    "published": "2018-10-08T13:29:00",
    "status": "Modified",
    "summary": "Python Cryptographic Authority pyopenssl version Before 17.5.0 contains a CWE - 401 : Failure to Release Memory Before Removing Last Reference vulnerability in PKCS #12 Store that can result in Denial of service if memory runs low or is exhausted. This attack appear to be exploitable via Depends upon calling application, however it could be as simple as initiating a TLS connection. Anything that would cause the calling application to reload certificates from a PKCS #12 store.. This vulnerability appears to have been fixed in 17.5.0.",
    "cvss3": null,
    "cvss2": null,
    "cpes": [
        {
            "id": 102099
        },
        {
            "id": 102101
        },
        {
            "id": 102102
        },
        {
            "id": 119831
        },
        {
            "id": 188918
        },
        {
            "id": 188959
        },
        {
            "id": 188960
        },
        {
            "id": 209984
        },
        {
            "id": 573539
        },
        {
            "id": 583029
        },
        {
            "id": 1168462
        }
    ]
}